rosbag2_compression_zstd/src/rosbag2_compression_zstd/zstd_decompressor.cpp

```diff
@@ -45,8 +44,8 @@ ZstdDecompressor::~ZstdDecompressor()
 std::string ZstdDecompressor::decompress_uri(const std::string & uri)
 {
   const auto start = std::chrono::high_resolution_clock::now();
-  const auto uri_path = rcpputils::fs::path{uri};
-  const auto decompressed_uri = rcpputils::fs::remove_extension(uri_path).string();
+  auto uri_path = std::filesystem::path{uri};
+  const auto decompressed_uri = uri_path.replace_extension().generic_string();
```

Comment:
```suggestion
  auto decompressed_uri = uri_path.replace_extension().generic_string();
```
CLion showing a warning when we are trying to return `decompressed_uri` from method
> Clang-Tidy: Constness of 'decompressed_uri' prevents automatic move 

i.e. const prevents copy-elision for the returning value

Reply:
nice, fixed